src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+
+Beyond the security of the Ethereum network, which contentious forks may compromise, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
+## Handling disagreements {#disagreements}
+
+Having many stakeholders with different motivations and beliefs means that disagreements are not uncommon.
+
+Generally, disagreements are handled with long-form discussion to understand the root of the problem. Typically, one group concedes, or a happy medium is achieved. If one group feels strongly enough, implementing a particular change on Ethereum could result in a split chain.
+
+### The DAO fork {#dao-fork}
+
+The DAO fork was in response to the [2016 DAO attack](https://www.coindesk.com/understanding-dao-hack-journalists) where an insecure [DAO](/glossary/#dao) contract was drained of over 3.6 million ETH in a hack. The fork moved the funds from the faulty contract to a new contract allowing anyone who lost funds in the hack to recover them.
+
+This course of action was voted on by the Ethereum community. Any ETH holder was able to vote via a transaction on [a voting platform](http://v1.carbonvote.com/). The decision to fork reached over 85% of the votes.
+
+Some miners refused to fork because the DAO incident wasn't a defect in the protocol. They went on to form [Ethereum Classic](https://ethereumclassic.org/).
+
+Watch more on the DAO hack:
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/rNeLuBOVe8A" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<Divider />
+
+## Beacon Chain Development {#beacon-chain}
+
+The Ethereum governance process often trades off speed and efficiency for openness and inclusivity. In order to accelerate the development of the Beacon Chain, it was launched separately from the proof of work Ethereum network and followed its own governance practices.
+
+While the specification and implementations development has always been fully open source, the formal processes used to propose updates described above weren't used. This allowed changes to be specified and agreed upon quicker by researchers and implementers.
+
+When the Beacon Chain merges with the Ethereum execution layer, the governance process to propose changes will be harmonized.
+
+<ButtonLink to="/eth2/merge/">More on the merge</ButtonLink>
+
+<Divider />
+
+## How can I get involved? {#get-involved}
+
+- [Propose an EIP](/eips/#participate)
+- [Discuss current proposals](https://ethereum-magicians.org/)
+- [Get involved in R&D discussion](https://discord.gg/mncqtgVSVw)
+- [Run a node](/developers/docs/nodes-and-clients/)
+
```

Comment:
Contribute to client development / become a core developer?

Reply:
@samajammin any suggested place to link?

Reply:
@samajammin anywhere you would suggest to link out to? One place I can think of is the [core dev apprenticeship program ](https://blog.ethereum.org/2021/05/13/core-dev-apprenticeship/) (if they plan on continuing, which I think they are)

Reply:
Seems like cohort 1 applications [start on Monday](https://discord.com/channels/595666850260713488/843176491386667018/882419405559046214)

Reply:
I think linking to the projects might be adequate, i.e. https://ethereum.org/en/developers/docs/nodes-and-clients/#clients

All these projects are open source & welcome contributions.